search/lib/settings.js

```diff
@@ -54,7 +54,8 @@ function getSettings () {
     settings.cmrStacRelativeRootUrl = process.env.CMR_STAC_RELATIVE_ROOT_URL || '/stac';
     settings.cmrSearchHost = process.env.CMR_SEARCH_HOST || 'cmr.earthdata.nasa.gov/search';
     settings.cmrProviderHost = process.env.CMR_PROVIDER_HOST || 'cmr.earthdata.nasa.gov/ingest/providers';
-    settings.cmrSearchProtocol = process.env.CMR_SEARCH_PROTOCOL || 'https';
+    settings.cmrProtocol = process.env.CMR_PROTOCOL || 'https';
```

Comment:
Are there any potential issues with an X-Forwarded-Proto header from a load-balancer or proxy causing issues with the response or expected request type?

Reply:
I don't think so, this basically just makes it so that when you deploy you need to know what the server will support and set it accordingly.